plugins/topology/src/utils/resource-utils.ts

```diff
@@ -86,6 +91,16 @@ const getPodTemplate = (
       return resource as V1PodTemplate;
     case 'Deployment':
       return (resource as V1Deployment).spec?.template;
+    case 'StatefulSet':
+      return (resource as V1StatefulSet).spec?.template;
+    case 'Job':
+      return (resource as V1Job).spec?.template;
+    case 'CronJob':
+      return (resource as V1CronJob).spec?.jobTemplate?.spec?.template;
+    case 'ReplicaSet':
```

Comment:
@debsmita1 We are not visualising replicasets so do we need to add it here?

cc: @invincibleJai

Reply:
Had added it by mistake, my bad.
Fixed it